src/server/ReplicaService.d.ts

```diff
@@ -29,64 +15,54 @@ export interface ReplicaService {
 	 */
 	RemovedActivePlayerSignal: RBXScriptSignal<(player: Player) => void>;
 	/**
-	 * A replica that is not replicated to any player and a "helper" for creating nested `Replica` objects
-	 * when immediate replication of individual nested replicas is not desirable.
+	 * A replica that is not replicated to any player and a "helper" for creating nested `Replica` objects when immediate replication of individual nested replicas is not desirable.
+	 *
+	 * **ReplicaService.Temporary Example:** TODO
```

Comment:
This TODO should be removed, examples are found in the original ReplicaService documentation anyway